extensions/azurehybridtoolkit/notebooks/hybridbook/content/hadr/configure-failover.ipynb

```diff
@@ -380,26 +378,30 @@
         {
             "cell_type": "markdown",
             "source": [
-                "### **Validate cluster (done locally on server)**\n",
+                "<span style=\"font-size: 16.38px;\"><b>Validate Failover Cluster</b></span>\n",
```

Comment:
Remove custom formatting here and elsewhere. Use default markdown such as one hash mark (#) for Titles, two and more for Headings. Remove the \<b\> tags and specific font sizes for headings. It's inconsistent and unnecessary to bold headers